templates/ntp.conf.erb

```diff
@@ -1,9 +1,11 @@
 # ntp.conf: Managed by puppet.
 #
-<% if @panic == false -%>
-# Keep ntpd from panicking in the event of a large clock skew
-# when a VM guest is suspended and resumed.
-tinker panic 0
+<% if @tinker == true -%>
+# Enable next tinker options:
+# panic - keep ntpd from panicking in the event of a large clock skew
+# when a VM guest is suspended and resumed;
+# stepout - allow ntpd change offset faster
+tinker panic <%= @panic %><% if @stepout.to_i != 300 -%> stepout <%= @stepout %><% end %>
```

Comment:
If you want these options to only be set if not the default value, I'd rather see them default to undef rather than having the defaults hardcoded in the template.